packages/api-v2/src/hooks/usePaymentAgentList.ts

```diff
@@ -0,0 +1,22 @@
+import useQuery from '../useQuery';
+import useActiveAccount from './useActiveAccount';
+import useSettings from './useSettings';
+
+/** A custom hook to get paymentagent list */
+const usePaymentAgentList = () => {
+    const { data: settings } = useSettings();
+    const { data: activeAccount } = useActiveAccount();
+
+    const { data, ...rest } = useQuery('paymentagent_list', {
```

Comment:
at this stage, probably worth adding (unless we do not care about authorise status): 
<img width="538" alt="Screenshot 2024-03-14 at 16 42 27" src="https://github.com/binary-com/deriv-app/assets/141034155/c5637bd4-aae9-473a-b3d0-e6243b570412">

at this point in time, there is not authomatic authorise so it still requires manual check

Reply:
@wojciech-deriv since we are running `paymentagent_list` request based on `country_code` existence and our `country_code` is coming from `useSettings` hook which is under the hood has authorisation check i decided to skip authorisation check in usePaymentAgentList hook